packages/core/src/components/sections/BannerText/BannerText.tsx

```diff
@@ -3,10 +3,12 @@ import {
   BannerTextContentProps as UIBannerTextContentProps,
 } from '@faststore/ui'
 
-import { useOverrideComponents } from 'src/sdk/overrides/OverrideContext'
+import { useOverrideComponents } from '../../../sdk/overrides/OverrideContext'
```

Comment:
What is the advantage of changing the absolute path import to the relative one?

Reply:
The types work correctly when importing from a context outside `tsconfig.json`'s `include` directories. That's the case for starter.store, for example. Without this, this function would return `any` or incomplete types.